src/client/src/utils/window/openWindow.openfin.ts

```diff
@@ -71,7 +71,6 @@ export async function openWindow(
       content: [
         {
           type: "stack",
-          title: displayName || windowName,
```

Comment:
see https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/master/src/client/src/OpenFin/Window/ChildWindowFrame.tsx#L14-L15

not super intuitive, and not sure I understand the rationale behind the design of the API OF ChildWindowFrame, but this fix goes with the flow of how that component works today

Reply:
strange, but ok 👍🏻

Reply:
hmm - the ChildWindowFrame switching between windowName and windowTItle too .. not ideal, is it?